lms/templates/validation_error.html.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends 'templates/base.html.jinja2' %}
+
+{% set title = error.message %}
+
+{% block content %}
+  <img src="{{ 'lms:static/images/sad-annotation.svg'|static_path }}">
+  <p>{% trans %}Hypothesis received an invalid request.
+  There were problems with these request parameters:{% endtrans %}</p>
+  <ul>
+  {% for field in error.messages %}
+    <li>
+      {{ field }}:
+      <ul>
+        {% for message in error.messages[field] %}
+          <li>{{ message }}</li>
+        {% endfor %}
+      </ul>
+    </li>
```

Comment:
This is my least favorite thing about webargs so far: validation error messages are a dict of dicts of lists of strings, that need to be looped over like this. I think the format and structure may vary for different types of validation failure too. For now, this works for the validation errors that we have so far